packages/core/src/components/Combobox/components/ComboboxItems/ComboboxItems.tsx

```diff
@@ -128,17 +130,19 @@ export const ComboboxItems: React.FC<ComboboxItemsProps> = forwardRef(
           className={cx(styles.optionsContainer, className)}
           items={options}
           itemRenderer={createItemElementRenderer}
-          role="treegrid"
+          role="listbox"
           scrollableClassName={styles.scrollableContainer}
           onItemsRendered={onItemsRender}
           style={style}
+          id={id}
         />
       );
     } else {
       itemsElements = (
         <div
           className={cx(styles.scrollableContainer, styles.optionsContainer, className)}
-          role="treegrid"
+          role="listbox"
```

Comment:
I wonder if we have a case where combobox items are collapsible
if so, we should allow passing custom role, or have a boolean that should indicate whether it is a list or a tree

I think the change here is correct, I just wonder about the breaking change